python/lsst/meas/base/forcedPhotCoadd.py

```diff
@@ -50,6 +50,7 @@ def setDefaults(self):
         self.measurement.copyColumns["id"] = "id"
         self.measurement.copyColumns["parent"] = "parent"
         self.references.removePatchOverlaps = False  # see validate() for why
+        self.measurement.plugins.names |= ['base_InputCount', 'base_Variance']
```

Comment:
why do these get promoted to always be run, as opposed to camera configs or run specific configs? Is this something we thing everyone will want all the time?

Reply:
Certainly the `InputCount` is something everyone will want to use (e.g., it can be used as a poor-man's detection limit). HSC experience is that the `Variance` is helpful too.